packages/ods/src/components/drawer/tests/navigation/ods-drawer.e2e.ts

```diff
@@ -32,5 +33,38 @@ describe('ods-drawer navigation', () => {
       await page.waitForChanges();
       expect(await getFocusTagName()).toBe('INPUT');
     });
+
+    it('should focus on focusable element in the drawer with a trigger button', async() => {
+      await setup(`
+        <button id="button-trigger-drawer">Trigger Drawer</button>
+        <ods-drawer>
+          <button>Default button</button>
+          <input name="input"></input>
+        </ods-drawer>`);
+
+      await page.evaluate(() => {
+        const drawer = document.querySelector('ods-drawer') as HTMLElement & OdsDrawer| null;
+        document.querySelector('#button-trigger-drawer')?.addEventListener('click', () => {
+          if (drawer?.isOpen) {
+            drawer.close();
+          } else {
+            drawer?.open();
+          }
+        });
+      });
+
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Enter');
+      await page.waitForChanges();
+
+      await page.keyboard.press('Tab');
+      await page.waitForChanges();
+
+      expect(await getFocusTagName()).toBe('BUTTON');
```

Comment:
would be nice to differentiate this "button" from the trigger "button" to be sure that we focus the right one